smsdk/client.py

```diff
@@ -210,24 +210,6 @@ def get_parts(self, normalize=True, clean_strings_in=True, clean_strings_out=Tru
         df = self.get_data_v1('part_v1', 'get_parts', normalize, *args, **kwargs)
 
         return df
-
-
-    @ClientV0.get_machine_schema_decorator
```

Comment:
By removing the decorator, you are changing the return type from a dataframe to a tuple. This might break some users' code.